### DISCUSSION TITLE
AIO port is not correctly set when launching containers

### DISCUSSION THREAD

**Discussion by drzewiec** [nextcloud/all-in-one#5121] [Questions] [answered]
### Steps to reproduce
1. Start AIO, I am using docker-compose if that matters. Use port 8080 for AIO interface
2. Set up nginx reverse proxy to proxy https://domain:8443 to https://127.0.0.1:8080
3. Start containers using button from AIO interface

### Expected behavior 
The "Open Nextcloud AIO interface" button in the Nextcloud admin settings should point to https://domain:8443

### Actual behavior 
The button points to https://domain (no port set)

### Host OS 
Debian 12

#### Nextcloud AIO version 
9.4.1

#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the nginx config here that you used for the aio interface?
